sunpy/net/vso/vso.py

```diff
@@ -921,3 +921,49 @@ def unknown_method(self, response):
     @classmethod
     def _can_handle_query(cls, *query):
         return all([x.__class__.__name__ in core_attrs.__all__ + attrs.__all__ for x in query])
+
+    @classmethod
+    def register_values(cls):
+        try:
+            adict = cls.get_vso_values()
+        except URLError:
+            #            log.warn('Looks like the VSO server is not reachable.')
+            adict = {}
+        return adict
+
+    def get_vso_values():
+        """
+        Reads the VSO keywords and returns a dict with them for the attrs we support.
+        """
+
+        import urllib
+        import json
+        url = "https://vso1.nascom.nasa.gov/cgi-bin/registry_json.cgi"
+        headers = {"Content-Type": "application/x-www-form-urlencoded"}
+        data = {'fields': '["physobs"]'}
+        uedata = urllib.parse.urlencode(data)
+        edata = uedata.encode('ascii')
+
+        req = urllib.request.Request(url=url, data=edata, headers=headers)
+        response = urllib.request.urlopen(req)
+        json.loads(response.read())
```

Comment:
```suggestion
        return json.loads(response.read())
```

Reply:
this was just all here so i dont forget. this isnt even implemented.